sklearn/feature_selection/_mutual_info.py

```diff
@@ -31,8 +31,8 @@ def _compute_mi_cc(x, y, n_neighbors):
     Returns
     -------
     mi : float
-        Estimated mutual information. If it turned out to be negative it is
-        replace by 0.
+        Estimated mutual information in nats (natural logarithm). If it
+        turned out to be negative it is replaced by 0.
```

Comment:
Having "natural logarithm" is confusing since this is not the meaning of `nat` (i.e. natural unit of information).

```suggestion
        Estimated mutual information in nat units. If it
        turned out to be negative it is replaced by 0.
```

Could you apply the same changes on the other occurrences.

Reply:
I'm not sure I understand your point, from the correpsonding [wikipedia](https://en.wikipedia.org/w/index.php?title=Nat_(unit)&oldid=1180577334) page: "The natural unit of information (symbol: nat),[[1]](https://en.wikipedia.org/w/index.php?title=Nat_(unit)&oldid=1180577334#cite_note-IEC_80000-13:2008-1) sometimes also nit or nepit, is a unit of [information](https://en.wikipedia.org/wiki/Information) or [information entropy](https://en.wikipedia.org/wiki/Information_entropy), based on [natural logarithms](https://en.wikipedia.org/wiki/Natural_logarithms) and powers of [e](https://en.wikipedia.org/wiki/E_(mathematical_constant))"

Saying that the entropy estimated using a natural logarithm is equivalent to saying that the result is expressed in nats, but I think users might be more familiar with the notion of logarithm base rather that the "nat" unit, hence the precision.

Reply:
This is 2 different information. One is the definition that uses the natural logarithm, the other one is about providing the unit.

If we are going to provide the definition, then the user guide is a better place to define the mathematical details.